packages/server/lib/browsers/cri-client.ts

```diff
@@ -1,10 +1,13 @@
+import Bluebird from 'bluebird'
 import debugModule from 'debug'
 import _ from 'lodash'
 
 const chromeRemoteInterface = require('chrome-remote-interface')
+
 const debugVerbose = debugModule('cypress-verbose:server:browsers:cri-client')
-const debugVerboseSend = debugModule('cypress-verbose:server:browsers:cri-client:[-->]')
-const debugVerboseReceive = debugModule('cypress-verbose:server:browsers:cri-client:[<--]')
+const debugVerboseSend = debugModule('cypress-verbose:server:browsers:cri-client:send')
+const debugVerboseReceive = debugModule('cypress-verbose:server:browsers:cri-client:recv')
```

Comment:
```
cypress-verbose:server:browsers:cri-client:send:[-->]
cypress-verbose:server:browsers:cri-client:recv:[<--]
```

Reply:
Neither `DEBUG=cypress-verbose:server:browsers:cri-client:send:*` nor `DEBUG=cypress-verbose:server:browsers:cri-client:send` produce output, and `DEBUG=cypress-verbose:server:browsers:cri-client:send:[-->]` breaks `debug`: 

```
$ DEBUG=cypress-verbose:server:browsers:cri-client:send:[-->] npm run test-e2e -- --spec 1_base_url_spec.coffee
/home/flotwig/Projects/cypress/cypress3/node_modules/snapdragon/node_modules/debug/src/debug.js:153
      exports.names.push(new RegExp('^' + namespaces + '$'));
                         ^

SyntaxError: Invalid regular expression: /^cypress-verbose:server:browsers:cri-client:send:[--$/: Range out of order in character class
    at new RegExp (<anonymous>)
    at Function.enable (/home/flotwig/Projects/cypress/cypress3/node_modules/snapdragon/node_modules/debug/src/debug.js:153:26)
    at Object.<anonymous> (/home/flotwig/Projects/cypress/cypress3/node_modules/snapdragon/node_modules/debug/src/node.js:248:9)
    at Module._compile (internal/modules/cjs/loader.js:759:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:770:10)
    at Module.load (internal/modules/cjs/loader.js:628:32)
    at Function.Module._load (internal/modules/cjs/loader.js:555:12)
    at Module.require (internal/modules/cjs/loader.js:666:19)
    at require (internal/modules/cjs/helpers.js:16:16)
    at Object.<anonymous> (/home/flotwig/Projects/cypress/cypress3/node_modules/snapdragon/node_modules/debug/src/index.js:9:20)
```